mathjax3-ts/output/chtml/Wrappers/mtable.ts

```diff
@@ -30,6 +30,49 @@ import {MmlMtable} from '../../../core/MmlTree/MmlNodes/mtable.js';
 import {StyleList} from '../CssStyles.js';
 import {DIRECTION} from '../FontData.js';
 
+
+/*
+ * The heights, depths, and widths of the rows and columns
+ * Plus the natural height and depth (i.e., without the labels)
+ * Plus the label column width
+ */
+export type TableData = {
+    H: number[];
+    D: number[];
+    W: number[];
+    NH: number[];
+    ND: number[];
+    L: number;
+};
+
+/*
+ * Sum the elements of an array
+ */
+function SUM(A: number[]) {
```

Comment:
Well, it probably comes from the fact that some of my earliest programming was in C, and the compiler constants and macros were in all caps.  Small utility functions like these feel like those compiler macros to me, so I do them in all caps out of habit.

I will change them, and move them to a utility file so that they can be reused more easily.